.circleci/config.yml

```diff
@@ -189,71 +206,55 @@ commands:
               else
                 echo "Need to delete branch manually"
               fi  
-  nofity_success:
+  nofity-success:
     steps:
       - run:
           name: "notify slack when job success"
           command : |
-            curl --data '{"text": ""✅ Job **'$CIRCLE_JOB'** succeeded on **'$CIRCLE_BRANCH''$CIRCLE_TAG'**. Please see '$CIRCLE_BUILD_URL' for details."}' "$SLACK_WEBHOOK_URL"
+            curl --data '{"text": "✅ Job **'$CIRCLE_JOB'** succeeded on **'$CIRCLE_BRANCH''$CIRCLE_TAG'**. Please see '$CIRCLE_BUILD_URL' for details."}' "$SLACK_WEBHOOK_URL"
           when: on_success
-  nofity_failure:
+  nofity-failure:
     steps:
       - run:
           name: "notify slack when job fail"
           command : |
-            curl --data '{"text": ""❌ Job **'$CIRCLE_JOB'** failed on **'$CIRCLE_BRANCH'**. Please see '$CIRCLE_BUILD_URL' for details."}' "$SLACK_WEBHOOK_URL"
+            curl --data '{"text": "❌ Job **'$CIRCLE_JOB'** failed on **'$CIRCLE_BRANCH'**. Please see '$CIRCLE_BUILD_URL' for details."}' "$SLACK_WEBHOOK_URL"
           when: on_fail
 
 jobs:
   build:
     machine: true
     working_directory: ~/go/src/github.com/klaytn/klaytn
     steps:
-      - checkout
+      - git-clone
+      - install-golang
```

Comment:
Do we need to install golang every time we build?

Reply:
Cause golang version installed on circleCI machine is v1.9.3. I want all image run with go v1.13.3

Reply:
What about installing go v1.13.3 on the image?

Reply:
We installed go v1.13.3 on docker image(such as build_base, rpmbuild) but circleci machine. We can run EC2 machine with go v1.13.3 instead of circleci machine. but it need a bit more time

Reply:
got it. Please ignore my comment.